x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/rest/RestSqlTestCase.java

```diff
@@ -97,10 +98,10 @@ public void testNextPage() throws IOException {
         for (int i = 0; i < 20; i += 2) {
             Map<String, Object> response;
             if (i == 0) {
-                response = runSql(mode, new StringEntity(sqlRequest, ContentType.APPLICATION_JSON));
+                response = runSql(new StringEntity(sqlRequest, ContentType.APPLICATION_JSON), "");
             } else {
-                response = runSql(mode, new StringEntity("{\"cursor\":\"" + cursor + "\"}",
-                        ContentType.APPLICATION_JSON));
+                response = runSql(new StringEntity("{\"cursor\":\"" + cursor + "\"" + mode(mode) + "}",
+                        ContentType.APPLICATION_JSON), "");
```

Comment:
"" could be `StringUtils/Strings.EMPTY`

Reply:
I've added an EMPTY constant to `org.elasticsearch.xpack.sql.proto.StringUtils`. It seems there is no other class visible to `qa` that has this constant defined.